.cicd/test.sh

```diff
@@ -5,13 +5,17 @@ set -eo pipefail
 # tests
 if [[ $(uname) == 'Darwin' ]]; then # macOS
     set +e # defer error handling to end
-    source ~/.bash_profile && "./$@"
+    [[ "$CI" == 'true' ]] && source ~/.bash_profile
```

Comment:
One thing to note is that macOS defaults to using zsh now. Not sure if that'll be an issue in the future

Reply:
Good point! As long as they don't break from Unix convention, [this line](https://github.com/EOSIO/eos/blob/d75a5f19e9560e85c306e46ba44b5ef554d777c4/.cicd/test.sh#L1) should direct macOS to run this script in BASH. However, I am concerned about them removing `bash` in the future.